src/app/contentaccess/subscription/individual/bucket/bucket.html

```diff
@@ -0,0 +1,150 @@
+<div ng-controller="BucketController">
+	<div class="row">
+	  <div
+		class="col-xs-12"
+		style="float: none; margin-left: auto; margin-right: auto;"
+	  >
+		<form ng-submit="validateAndSubmit() && next('pay')">
+		  <div class="row" style="height: 55px;"></div>
+		  <div class="row">
+			<div class="col-xs-12 info-individual-heading">
+			  Bucket Purchase academic licenses for individual use
+			</div>
+		  </div>
+		  <div class="row">
+			<div class="col-xs-12 info-individual-text">
+			  Check
+			  <a
+				href="#contentaccess/list?partnerId={{partnerId}}"
+				target="_blank"
+				>this list</a
+			  >
+			  to see if you already have access to {{partner.name}} through your
+			  library. If your institution is subscribed and you are unable to
+			  access a page, please contact us at
+			  <a href="mailto:info@phoenixbioinformatics.org"
+				>info@phoenixbioinformatics.org</a
+			  >
+			</div>
+		  </div>
+		  <div class="row" style="height: 60px;"></div>
+		  <div class="row">
```

Comment:
This part, since the bucket pricing section is very long and therefore the display is folded, I think we do not need to arrange the "title" and the pricing side-by-side. They could each have their row.

Reply:
Kartik's suggestion: We should have somewhere explaining what is the bucket system and the units.
We need to put it on the marketing site eventually https://phoenixbioinfo.org/tair/, so this could be linked to that marketing site.
Also breaking news items and banner are expected.